### DISCUSSION TITLE
Can an async logger be used in different threads?

### DISCUSSION THREAD

**Discussion by Mdp11** [gabime/spdlog#2394] [Q&A]
Hello everyone,

one thing that is not clear to me is if async loggers are thread safe.

I'm constructing a logger in this way:
`auto logger = std::make_shared<spdlog::async_logger>(logger_name, sinks.begin(), sinks.end(), spdlog::thread_pool(), spdlog::async_overflow_policy::block);`

Can I use the same logger in different threads?

**Comment by tt4g** [comment]
Yes. The async-logger's logging API is thread-safe. However, remember to use thread-safe sinks (the `_mt` suffix sink is thread-safe).
